.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+
+<!--
+Thank you for opening a pull request!
+
+Please give your pull request a title like "[component]: [short description]"
+
+Please make sure all your git commit messages include a "Signed-off-by" line,
+and a "Fixes" line as appropriate.  This is the format for
+commit messages:
+
+"""
+[component]: [short description]
+
+[A longer multiline description]
+
+Fixes: [ticket URL on tracker.ceph.com]
+Signed-off-by: [Your Name] <[your email]>
+"""
+
+The component is the short name of a major daemon or subsystem,
+something like "mon", "osd", "mds", "rbd, "rgw", etc. For ceph-mgr modules,
+give the component as "mgr/<module name>" rather than a path into pybind.
+
+For more examples, simply use "git log" and look at some historical commits.
+
+If this is a bug fix, and a ticket does not yet exist on tracker.ceph.com,
+please create one now and reference it from your commit message.  Even for
+small or obvious fixes, the tracker ticket is important to enable
+the backporting process to stable branches.
+
+More information for contributors is available here:
```

Comment:
I would move this reference further up, directly after the "Thank you!" line. Then I would start the following part with "In a nutshell, this is what you should do:"